presto-docs/src/main/sphinx/connector/blackhole.rst

```diff
@@ -9,17 +9,17 @@ for data reading. However, it also has some other features that allow testing Pr
 in a more controlled manner. Metadata for any tables created via this connector
 is kept in memory on the coordinator and discarded when Presto restarts.
 Created tables are by default always empty, and any data written to them
-will be ignored and data reads will return no rows.
+is ignored and data reads return no rows.
 
 During table creation, a desired rows number can be specified.
-In such case, writes will behave in the same way, but reads will
-always return specified number of some constant rows.
+In such cases, writes behave in the same way, but reads
```

Comment:
why plural? Previous has singular "During table creation"

Reply:
There is no singular usage of this phrase in English. You could say

In this case

or 

In such cases 

So here we might want to change to "In this case, writes..."